### PR TITLE
[ADF-4550] Form Cloud - Be able to upload a file on the content taken from the process-storage

### PR DESCRIPTION
**Pull request by mauriziovitale**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
The file is uploaded based on the hostECM

**What is the new behaviour?**
The content is picked from the process-storage service

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-4550

### CODE REVIEW

File: lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts [hunk 1]
```diff
@@ -236,7 +238,7 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
     }
 
     async getFolderTask(appName: string, taskId: string) {
-        this.nodeId = await this.formCloudService.getProcessStorageFolderTask(appName, taskId).toPromise();
+        this.processStorageCloudModel = await this.formCloudService.getProcessStorageFolderTask(appName, taskId).toPromise();
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts]: the method is called "getFolderTask" but it does not return anything

Comment (mauriziovitale) [reply lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts]: Thanks @DenysVuika  I will fix it

File: lib/process-services-cloud/src/lib/form/services/form-cloud.service.ts [hunk 2]
```diff
@@ -89,18 +89,19 @@ export class FormCloudService extends BaseCloudService {
         );
     }
 
-    createTemporaryRawRelatedContent(file, nodeId): Observable<any> {
+    createTemporaryRawRelatedContent(file, nodeId, contentHost): Observable<any> {
```
Comment (eromano) [inline lib/process-services-cloud/src/lib/form/services/form-cloud.service.ts]: can we add the types?

Comment (mauriziovitale) [reply lib/process-services-cloud/src/lib/form/services/form-cloud.service.ts]: for now, the type is not used and is always alfresco

Comment (mauriziovitale) [reply lib/process-services-cloud/src/lib/form/services/form-cloud.service.ts]: @eromano let me know if sounds good

**Pull request merged**